patches/aesthetics.py

```diff
@@ -309,7 +308,49 @@ def reduceMessageLengths(rom, rnd):
         "seagull noises",
         "I'm telling you, YOU HAVE UNO, it came free with your Xbox!",
         "LADXR - Now with even more Marin quotes!",
-        "You guys are spending more time adding Marin quotes than actually playing the game."
+        "You guys are spending more time adding Marin quotes than actually playing the game.",
+        "NASA faked the moon.",
+        "Doh, I missed!",
+        "Beginning the seed in... 100\n99\n98\n97\n96\n...\nJust Kidding.",
+        "Consider libre software!",
+        "Consider a GNU/Linux installation!",
+        "Now you're gonna tell me about how you need to get some instruments or maybe shells to hatch a whale out of an egg, right?  All you boys are the same...",
+        "Oh hey #####!  I made pancakes!",
+        "Oh hey #####!  I made breakfast!",
+        "Alright Tarin, test subject number 142857 was a failure, give him the item and the memory drug and we'll try next time.",
+        "Betcha 100 rupees that Tarin gives you a sword.",

```

Comment:
You can use `{SWORD}` to get the name of items, so they get replaced if you select certain graphics sets (meme will call it the stick)